contracts/DankFaucet.sol

```diff
@@ -104,4 +108,20 @@ contract DankFaucet is usingOraclize {
    * Simple function to allow for adding ETH to the contract.
    */
   function sendEth() public payable { }
+
+  /**
+   * Allow the owner to reset the DANK we've allocated to a phone number.
+   */
+  function resetAllocatedDankForPhoneNumber(bytes32 hashedPhoneNumber, uint dank) auth {
+    uint dankInWei = (dank * 1000000000000000000);
```

Comment:
I think is better if the contract api accepts all values in wei, and the contracts only works in wei, so no conversion or weird multiplications needed in the contract.

Reply:
I stuck with the convention in the rest of the contract to take values in and multiply to get wei. Do you think I should remove this in both places in the contract?

Reply:
Yeah I think it is better to have everything in wei inside contracts.

Reply:
yes, everything in wei inside contracts please